Avoid superfluous handling for X11/Wayland with gtk+, warn in pkg_postinst intead.
--- a/src/libaudgui/init.cc
+++ b/src/libaudgui/init.cc
@@ -342,7 +342,7 @@ EXPORT void audgui_init ()
     if (init_count ++)
         return;
 
-#if defined(GDK_WINDOWING_WAYLAND) && defined(GDK_WINDOWING_X11)
+#if 0 /// Avoid handling X11/Wayland just for that. Warn in pkg_postinst instead
     // Use X11/XWayland by default, but allow to overwrite it.
     // Especially the Winamp interface is not usable yet on Wayland
     // due to limitations regarding application-side window positioning.
